docs/installation.md

```diff
@@ -129,6 +131,26 @@ Save the file. You'll need to open a new shell/terminal window for the changes t
 
 {% /details %}
 
+{% details summary="Windows" %}
+First, determine if the bun binary is properly installed on your system:
+
+```pwsh
+& "$env:USERPROFILE\.bun\bin\bun" --version
+```
+
+If the command executes successfully but `bun --version` is not recognized, it means bun is not in your system's `PATH`. To add it, open an elevated (as administrator) Powershell and paste the following command:
+
+```pwsh
+[System.Environment]::SetEnvironmentVariable(
+    "Path",
+    [System.Environment]::GetEnvironmentVariable("Path", "Machine") + ";$env:USERPROFILE\.bun\bin",
+    [System.EnvironmentVariableTarget]::Machine
```

Comment:
Please use `[System.EnvironmentVariableTarget]::User`, this way it does not need an Admin prompt.

https://learn.microsoft.com/en-us/dotnet/api/system.environmentvariabletarget?view=net-9.0